code-of-conduct (#9)
blitz-js/fr.blitzjs.com: app/pages/docs/code-of-conduct.mdx

---
id: code-of-conduct
title: Le Code de Conduite de la Communauté Blitz
sidebar_label: Code de Conduite
---

<!--alex disable racist sexual nuts-->

Les membres principaux de Blitz prennent très au sérieux ce Code de Conduite (CoC). Tous les membres,
contributeurs et volontaires dans cette communité sont priés d'agir

> en fonction du Code de Conduite qui suit afin que Blitz continue de grandir, que la communauté garde un cadre positif, et qu'ils nous aident à apporter et assurer un environnement

Reviewer: ```suggestion
en fonction du Code de Conduite qui suit, afin que Blitz continue de grandir, que la communauté garde un cadre positif, et qu'ils nous aident à apporter et assurer un environnement
```

---
id: code-of-conduct
title: Le Code de Conduite de la Communauté Blitz
sidebar_label: Code de Conduite
---

<!--alex disable racist sexual nuts-->

Les membres principaux de Blitz prennent très au sérieux ce Code de Conduite (CoC). Tous les membres,
contributeurs et volontaires dans cette communité sont priés d'agir

> en fonction du Code de Conduite qui suit pour préserver Blitz afin que ce projet grandissant
> et la communauté gardent un cadre positif, et qu'ils nous aident à apporter et assurer un environnement
> sain pour tout le monde.

Reviewer 2: ```suggestion
en fonction du Code de Conduite qui suit afin que Blitz continue de grandir, que la communauté garde un cadre positif, et qu'ils nous aident à apporter et assurer un environnement
sain pour tout le monde.
```

> ## Quand Quelque Chose Arrive {#when-something-happens}

Reviewer 2: ```suggestion
## Si Vous Êtes Témoin d'un Incident {#when-something-happens}
```

Si vous voyez une violation du Code de Conduite, suivez ces étapes :

> 1. Faites savoir à la personne que ce qu'elle a fait n'est pas approprié et demandez-lui
>    d'arrêter et/ou d'éditer leur(s) message(s).

Reviewer 2: ```suggestion
1. Faites savoir à la personne que ce qu'elle a fait n'est pas approprié et demandez-lui
   d'arrêter et/ou de corriger leur(s) message(s).
```

> 2. Cette personne devra immédiatement stopper ce comportement et remédier le problème.

Reviewer 2: ```suggestion
2. Cette personne devra immédiatement arrêter ce comportement et remédier au problème.
```

3. Si ce n'est pas le cas, ou si vous ne vous sentez pas à l'aise à vous exprimer, contactez
   Brandon Bayer ([Twitter](https://twitter.com/flybayer) |
   [Email](mailto:b@bayer.ws)).

> Lors du report, veuillez inclure autant de détails, de liens, de captures d'écran,

Reviewer 2: ```suggestion
Lors de votre rapport, veuillez inclure autant de détails, de liens, de captures d'écran,
```

  tout ceci ou de leurs attributs.
- Nous sommes plus forts ensemble. Nous nous ressemblons plus que nous sommes différents.
- Notre communauté est basée sur le respect mutuel, la tolérance, et l'encouragement.
- Nous croyons qu'une communauté diverse où les personnes traitent chacun avec
  respect est plus forte, plus vibrante et possède plus de contributeurs potentiels
  et plus de ressources et d'idées. Nous visons à avoir plus de diversité.
- Nous sommes gentils, accueillants et courtois envers tous.
- Nous sommes respectueux des autres, de leur position, de leurs compétences, de leurs
  engagements et de leurs efforts.
- Nous sommes attentifs dans nos communications, que ce soit en personne ou en ligne, et
  nous sommes prudents lors d'échanges de différents points de vue.
- Nous sommes conscients que le langage façonne la réalité. Ainsi, nous comprenons
  un langage neutre dans les documents que nous procurons et lors de conversations avec
  autrui. Lorsque nous faisons référence à un groupe de personnes, nous visons à utiliser
  des termes neutres tels que “équipe”, “gens”, “tout le monde”. (Pour plus d'informations, nous
  recommandons

>   [ce post](https://modelviewculture.com/pieces/gendered-language-feature-or-bug-in-software-documentation)).

Reviewer 2: ```suggestion
[cette publication](https://modelviewculture.com/pieces/gendered-language-feature-or-bug-in-software-documentation)).
```

- Nous respectons le fait que tout le monde possède des opinions différentes et puissent en débattre
  de manière constructive.

> - Nous estimons les personnes à travers le code.

Reviewer 2: ```suggestion
- Les individus sont plus importants que le code.
```

  intention d'arnaquer ne sont pas les bienvenus.
- Les sponsors de Blitz sont tout aussi sujets à ce Code de Conduite.
  Notamment, les sponsors se doivent de ne pas utiliser d'images à caractère sexuel,
  ou d'autre matériel qui ne seraient pas en accord avec ce Code de Conduite.

## Les Conséquences aux Violations de ce Code de Conduite {#consequences-for-violations-to-this-code-of-conduct}

Si un participant adopte un comportement violant ce Code de Conduite,
les membres principaux de cette communauté prendront les mesures nécessaires,
commençant par un simple avertissement, escaladant au besoin
jusqu'à l'expulsion de la communauté, l'exclusion de toute forme d'interactions et la perte
de tous droits au sein de la communauté.

## Les Décisions concernant les Conséquences des Violations {#decisions-about-consequences-of-violations}

Les décisions concernant les conséquences à la suite de violations de ce Code de Conduite sont
prises par les membres principaux de cette communauté et peuvent ne pas en être discuté avec le
principal responsable de la violation.

## Pour toute Question ou Feedback {#for-questions-or-feedback}

> Si vous avez une question ou un feedback concernant ce Code de Conduite, nous sommes heureux

Reviewer 2: ```suggestion
Si vous avez une question ou une suggestion concernant ce Code de Conduite, nous sommes heureux
```
